### DISCUSSION TITLE
Fortinet Routing

### DISCUSSION THREAD

**Discussion by TwistyTwistedTwister** [Vadims06/topolograph#51] [Ideas]
Would it be possible to add Fortinet products to topolograph?
Commands include:
get router info ospf database network lsa
get router info ospf database external lsa
get router info ospf database router lsa

**Comment by Vadims06** [comment]
Hi @TwistyTwistedTwister ,
Yeah, I will be glad to add Fortinet support if I have a same output from these commands. Could you please send it to admin at topolograph.com? Feel free to mimic public IP addresses if they exist in the outputs.

**Comment by plamend** [comment]
Hi Could you add Fortinet to the list. Thank you.

**Reply by Vadims06** [reply]
Hi @plamend ,
I haven't received the command' outputs from the requestor, that's why Fortinet hasn't been onboarded. Could you please send it to me? Feel free to replace public IPs in the output if they exist into it.

**Comment by plamend** [comment]
Hi Thank you for the reply.
I am interested to deploy the tool in docker mode.
Is the process with new vendor on boarding the same ?


На пн, 13.01.2025 г. в 22:13 Vadim ***@***.***> написа:

> Hi @plamend <https://github.com/plamend> ,
> I haven't received the command' outputs from the requestor, that's why
> Fortinet hasn't been onboarded. Could you please send it to me? Feel free
> to replace public IPs in the output if they exist into it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Vadims06/topolograph/discussions/51#discussioncomment-11824370>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABHSTK74J5SXMXVPWCBUIA32KQM7XAVCNFSM6AAAAABKJ6OLN6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCOBSGQZTOMA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Best Regards/Поздрави


*Plamen I Danov*

------------------------------
Phone: +359898895855
E-mail: ***@***.***

**Reply by Vadims06** [reply]
Yes, it's possible to create parsing templates yourself, the process is described here https://github.com/Vadims06/topolograph/wiki/How-to-add-new-vendor-support
Share such templates with me or push them into https://github.com/Vadims06/topolograph/tree/master/textfsm_templates.
Then I can build and release new docker container image version with these templates

**Comment by plamend** [comment]
Hi Vladim,I am working on an OSPF topology with a complicated design now.
Is there any documentation how the topolograph will represent NSSA in the
picture? It looks like on my project it is as a separate topology not
connected to the backbone.
Thank you .

На ср, 15.01.2025 г. в 1:47 Vadim ***@***.***> написа:

> Yes, it's possible to create parsing templates yourself, the process is
> described here
> https://github.com/Vadims06/topolograph/wiki/How-to-add-new-vendor-support
> Share such templates with me or push them into
> https://github.com/Vadims06/topolograph/tree/master/textfsm_templates.
> Then I can build and release new docker container image version with these
> templates
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Vadims06/topolograph/discussions/51#discussioncomment-11837071>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABHSTKYG5KHEQ7PX3T5UG6L2KWO2DAVCNFSM6AAAAABKJ6OLN6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCOBTG4YDOMI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Best Regards/Поздрави


*Plamen I Danov*

------------------------------
Phone: +359898895855
E-mail: ***@***.***

**Reply by Vadims06** [reply]
Sorry for the late reply,
So it's about joining LSDB from multi areas. It's possible to collect the output from Area Border Router and import it into topolograph or use Python API and include outputs from different routers (which sit in different areas) and the output will be merged in a single graph. Let me know if any help is needed here

**Comment by plamend** [comment]
Thank you. If the LSDBs are from a different ospf processes, can I upload
in one text file.and get the topology maps combined?

**Reply by Vadims06** [reply]
Yes, it should work

**Comment by Vadims06** [comment]
Hi @plamend , @TwistyTwistedTwister 
Thanks to @JasonCook599 Fortinet was onboarded to Topolograph, could you please check if graph is built correctly. 
Please pull https://github.com/Vadims06/topolograph-docker and replace `topolograph:latest` to `topolograph:2.43_dev`

**Reply by JasonCook599** [reply]
This is not currently working. Details mentioned in #63
